guides/fork-new-package.md

```diff
@@ -0,0 +1,19 @@
+# Forking a New Package
+
+This documentation details the criteria that should be used for determining if an existing package should be forked into the `pulsar-cooperative` organization.
+
+While making these choices, be very mindful of the reason this org exists:
+This org is intended to allow a package to continue it's useful lifespan after such a time that it would be impossible to do so otherwise. To make sure users and contributors have a place to report issues with a package where they will be heard, and have a place to make contributions to a package where they will be reviewed.
+
+Bearing this in mind, below are the loose guidelines that should be followed and considered when forking a community package:
+
+* Is this package archived? An archived package means that no new issues or PRs can be created, and this new repo may be the only place for the community to be heard.
+* Is this package abandoned? While difficult to determine, make sure to keep in mind more than the last commit date. Are there issues left unaddressed? PR's that have never been reviewed or commented on? How long has it been since this repo was ever interacted with by anyone with write access?
```

Comment:
```suggestion
* Is this package abandoned? While difficult to determine, make sure to keep in mind more than the last commit date. Are issues left unaddressed by maintainers? PR's that have had a long time since being reviewed or commented on, if ever? How long has it been since this repo was ever interacted with by anyone with write access?
```